src/promptflow-devkit/promptflow/_proxy/_python_executor_proxy.py

```diff
@@ -85,6 +85,14 @@ async def exec_aggregation_async(
     ) -> AggregationResult:
         return self._flow_executor.exec_aggregation(batch_inputs, aggregation_inputs, run_id=run_id)
 
+    async def exec_line_async(
```

Comment:
Where would it be used for?

Reply:
Yeah, "exec_line_async is called by the batch engine (see Ln 565 in _batch_engine.py). We're just doing some plumbing work here to enable the async batch run. This change is based on a PR that Heyi shared earlier: 

[[Batch] Enable async batch if environment is set by thy09 · Pull Request #3272 · microsoft/promptflow (github.com)](https://github.com/microsoft/promptflow/pull/3272)